src/main/java/com/softserve/messaging/model/MessageHandler.java

```diff
@@ -1,34 +1,32 @@
-package com.softserve.message_sender.domain;
+package com.softserve.messaging.model;
 
+import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import java.time.ZonedDateTime;
+import java.util.UUID;
 
 @Entity
 @Data
 @NoArgsConstructor
-public class UserReceipt {
+@AllArgsConstructor
+public class MessageHandler {
 
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
 
+    private UUID userUUID;
     private String firstName;
     private String lastName;
     private String mail;
     private double cost;
+    private String token;
+    private ZonedDateTime zonedDateTime;
```

Comment:
1. what kind of information does this field keep? 
2. why does it need zone? `Instant` is usually used for timestamps.